recipes/libsvtav1/all/conanfile.py

```diff
@@ -43,6 +43,10 @@ def configure(self):
     def layout(self):
         cmake_layout(self, src_folder="src")
 
+    def requirements(self):
+        if self.settings.arch in ("x86", "x86_64"):
+            self.requires("cpuinfo/cci.20231129")
```

Comment:
```suggestion
        self.requires("cpuinfo/cci.20231129")
```

I would not keep that restriction, because the cpuinfo package has no limitation. Plus, when using external cpuinfo in libsvtav1, there is no restriction about the arch too.

Reply:
Good point, thanks